File: benchmark/php/count_set_bits.php
<?php
$start = hrtime(true);
$total = 0;
for ($n = 1; $n <= 100000; $n++) {
    $x = $n;
    while ($x > 0) { if ($x % 2 == 1) $total++; $x = intdiv($x, 2); }
}
$end = hrtime(true);
$ms = ($end - $start) / 1e6;
echo "Total: " . $total . "
";
echo "Time: " . round($ms) . "ms
";

File: benchmark/src/python/count_set_bits.py
import time
n=100000;total=0;start=time.time()*1000
for i in range(1,n+1):
 x=i
 while x>0:total+=x%2;x//=2
end=time.time()*1000
print(f"Result: {total}")
print(f"Time: {int(end-start)}ms")